pyscraper/regmem/commons/api_models.py

```diff
@@ -41,23 +41,34 @@ def multi_to_camel(value: str) -> AliasChoices:
 
 interest_template = Template(
     """
-        <p class="interest-summary">{{ interest.summary|e  }}</p>
-
-        <ul class="interest-details-list">
-        {% for field in interest.present_fields() %}
-            {{field.to_html()}}
-        {% endfor %}
-            {% if interest.registration_date %}
-            <li class="registration-date">Registration Date: {{ interest.registration_date.strftime('%d %B %Y') }}</li>
-            {% endif %}
-            {% if interest.published_date %}
-            <li class="published-date">Published Date: {{ interest.published_date.strftime('%d %B %Y') }}</li>
-            {% endif %}
-            {% if interest.last_updated_date %}
-            <li class="last-updated-date">Last Updated Date: {{ interest.last_updated_date.strftime('%d %B %Y') }}</li>
+        <div class="interest-item" id="{{ interest.id }}">
+            <p class="interest-summary parent-interest-item">{{ interest.summary|e  }}</p>
+
+            <ul class="interest-details-list">
+            {% for field in interest.present_fields() %}
+                {{field.to_html()}}
+            {% endfor %}
+                {% if interest.registration_date %}
+                <li class="registration-date">Registration Date: {{ interest.registration_date.strftime('%d %B %Y') }}</li>
+                {% endif %}
+                {% if interest.published_date %}
+                <li class="published-date">Published Date: {{ interest.published_date.strftime('%d %B %Y') }}</li>
+                {% endif %}
+                {% if interest.last_updated_date %}
+                <li class="last-updated-date">Last Updated Date: {{ interest.last_updated_date.strftime('%d %B %Y') }}</li>
+                {% endif %}
+
+            </ul>
+            {% if interest.child_items %}
+            <p class="child-item-header">Specific work or payments</p>
```

Comment:
Sorry, could this be a heading if it's a heading

Reply:
Same for `interest-summary` if that's the heading? (if you're wanting to make it bold at the TWFY end)
Or is it a definition list, dt /dd, hmm.
I see the existing code was no better, so, whatever, just seemed bit odd.

Reply:
(i was avoiding this because it's weird either in the file or in the site, but better to be weird in the file).

Reply:
Ok! header structure added in latest commit.